spec-0014/index.md

```diff
@@ -0,0 +1,410 @@
+    <br>
+    3.1. [Configuration](#31-configuration)
+
+#### 1. Support the WebAssembly platform by adding a Pyodide CI job
+
+{{< admonition note >}}
+While this step is not required to enable interactive documentation deployments alone, it is recommended to ensure that projects work properly in the WebAssembly environments used for such deployments and to facilitate their maintenance. We recommend that both pure Python projects and those with compiled extensions do this.
+
+{{< /admonition >}}
+
+##### 1.1. How to assess compatibility with WebAssembly
+
+First, we discuss assessing whether a project is compatible with WebAssembly.
+
+While we note that Core Projects will need to consider the below points when aiming to establish compatibility with WebAssembly, it is worth mentioning that [most Core Projects are already available in the Pyodide distribution](https://pyodide.org/en/stable/usage/packages-in-pyodide.html), thanks to the work collectively done by the Pyodide maintainers, the emscripten-forge team, and several external collaborators.
+
+Two major projects are currently providing ports of CPython to a WebAssembly runtime through the Emscripten compiler toolchain: Pyodide and emscripten-forge. They are largely compatible with CPython similar to how it runs on a standard Linux machine, albeit with a few differences, limitations, and exceptions:
+
+- The bitness is 32-bit, with 64-bit memory indices left for future iterations of the WebAssembly specification
+- CPython is directly compiled to WASM, specifically, the `wasm32-unknown-emscripten` target triplet, and the Python virtual machine that runs Python programs itself runs inside browsers' WebAssembly virtual machine implementations. While the Emscripten compiler toolchain provides JavaScript glue code that integrates with web-based elements in browsers, POSIX standard compliance—while generally high—is implemented differently due to browsers' file system and memory sandboxing features.
+- This means that specific modules from the Python standard library are unavailable and have been removed, and there is a lack of support for the `threading` and `multiprocessing` modules. Core Projects relying on parallel programming functionality, for example, using pthreads or OpenMP must adapt their code accordingly.
+
+For more details, refer to [the Pyodide documentation on "Pyodide Python compatibility"](https://pyodide.org/en/stable/usage/wasm-constraints.html).
+
+##### 1.2. What does adding support for WebAssembly look like?
+
+First, Core Projects should check if this criterion is satisfied already, as [many Scientific Python projects are now well-tested for usage and interoperability in a WebAssembly runtime provided by Pyodide and emscripten-forge](https://github.com/Quansight-Labs/czi-scientific-python-mgmt/issues/18). This goal's accomplishment amplifies the notion of Scientific Python being runnable in browser-based runtimes.
```

Comment:
I think we can remove this sentence (I find it a bit hard to parse)
```suggestion
First, Core Projects should check if they are not already supported, as [many Scientific Python projects are now well-tested for usage and interoperability in a WebAssembly runtime provided by Pyodide and emscripten-forge](https://github.com/Quansight-Labs/czi-scientific-python-mgmt/issues/18).
```

Reply:
Sounds good! A question here: should it be "if they are already supported" or "if they are not already supported"?

I think the former is making more sense, as to me the sentence is saying that Core Projects should check if they already support WebAssembly, because many projects are now well-tested for it.

i.e., we're trying to say "Check if you already have WebAssembly support, because many projects do."

Reply:
Yes you are right, some of my portuguese escaped in there 😄

Reply:
Thanks! Updating this.